code/modules/antagonists/heretic/magic/apetravulnera.dm

```diff
@@ -16,14 +16,16 @@
 
 	cast_range = 4
 	/// What type of wound we apply
-	var/wound_type = /datum/wound/slash/flesh/critical/cleave
+	var/wound_type = WOUND_SLASH
```

Comment:
This has a unique subtype of wound that it applies (`/datum/wound/slash/flesh/critical/cleave`), please ensure that that type is being applied still with your changes

Reply:
oh i didnt notice that part, ugh